### ISSUE TITLE
update stylus icon

### ISSUE THREAD

**Issue by DanielPintilei** [jesseweed/seti-ui#309]
Something like this:
https://www.dropbox.com/s/d3bnz263dmz3xs6/stylus-logo.svg?dl=0
:+1:

**Comment by ErickPetru** [comment]
Agreed, it uses the same font and colors from the official Stylus website.
